src/extension/rn-extension.ts

```diff
@@ -202,7 +210,10 @@ function setupAndDispose<T extends ISetupableDisposable>(setuptableDisposable: T
 
 function isSupportedVersion(version: string): boolean {
     if (!!semver.valid(version) && !semver.gte(version, "0.19.0")) {
-        TelemetryHelper.sendSimpleEvent("unsupportedRNVersion", { rnVersion: version });
```

Comment:
This change looks redundant to me, because previous event is already being used.